assets/src/application/ui/input/Button/IconButton/style.scss

```diff
@@ -8,7 +8,7 @@
 		background: none;
 		border: 1px solid var(--ee-background-color) !important;
 		box-shadow: none;
-		color: var(--ee-default-text-color);
+		color: var(--ee-default-text-color) !important;
```

Comment:
Have used important here to override this:

![image](https://user-images.githubusercontent.com/1477453/80212983-2762bd00-8641-11ea-8eb1-766924aadc09.png)

Reply:
hmmm ... ok we should try to solve that another way, but we can do that in another PR

Reply:
Have used that `.ee-btn-base` namespace
https://github.com/eventespresso/event-espresso-core/pull/2729/commits/ab81b81f73f24d45eb27ff15866b290a017056a7
***
but, will improve further in context of `button` styles overlap in: https://github.com/eventespresso/event-espresso-core/issues/2725